### PR TITLE
Issue #3348 support subthemes in color module

### PR DESCRIPTION
**Pull request by hosef**
https://github.com/backdrop/backdrop-issues/issues/3348

### CODE REVIEW

**Comment by hosef** [comment]
This PR should fix most of the issues needed to resolve backdrop/backdrop-issues#3348.

Primary changes include:
- Importing settings from the base theme
- Get the list of css files from the theme info if it is not defined in the color settings
- Getting a list of files from the base theme if the base theme does not support the color module
- Changed the code that swaps out files on page load so it works with all of the sub arrays of the stylesheets array from the theme instead of just stylesheets[screen]
- Allowed themes to disable color shifting by omitting the 'blend_target' from the color settings

File: core/modules/color/color.module [hunk 1]
```diff
@@ -186,13 +185,19 @@ function color_css_alter(&$css) {
...
-          $stylesheet = file_get_contents($theme_path . '/' . $file);
-          _backdrop_build_css_path(NULL, base_path() . dirname($paths['source'] . $file) . '/');
-          $stylesheet = preg_replace_callback('/url\([\'"]?(?![a-z]+:|\/+)([^\'")]+)[\'"]?\)/i', '_backdrop_build_css_path', $stylesheet);
+          $filename = _color_get_file_path($file, $preview_theme);
+          $stylesheet = file_get_contents($filename);
+          $paths['source'] = backdrop_get_path('theme', _color_get_theme_from_file($file, $preview_theme));
+          $base = base_path() . dirname($filename) . '/';
+          _backdrop_build_css_path(NULL, $base);
+
+          $paths['source'] = backdrop_get_path('theme', _color_get_theme_from_file($file, $preview_theme));
```
Comment (herbdool) [inline core/modules/color/color.module]: This line is repeated twice. Here and on line 190. Mistake?

Comment (hosef) [reply core/modules/color/color.module]: Ahh, yes. I added a new commit that removes the extra line.

**Comment by quicksketch** [comment]
Merged into 1.x and 1.15.x. Thanks!

**Pull request merged**